knexfile.js

```diff
@@ -1,10 +1,10 @@
 module.exports = {
     client: "mysql",
     connection: {
-        host: process.env.MYSQL_HOSTNAME,
-        port: process.env.MYSQL_PORT,
-        user: process.env.MYSQL_USERNAME,
-        password: process.env.MYSQL_PASSWORD,
-        database: process.env.MYSQL_DB
+        host: process.env.MYSQL_HOSTNAME || "localhost",
```

Comment:
Was this change a workaround for missing env vars when running migrations? It should be resolved now so long as you have the values defined in your `.env`

Reply:
Yeah it seems like you can't run the seeds without it, but I could be mistaken.